### PR TITLE
adds remarks when zero or null for IFindFluent.Limit 

### PR DESCRIPTION
**Pull request by leandromoh**

### CODE REVIEW

**Comment by leandromoh** [comment]
FYI @DmitryLukyanov @rstam

File: src/MongoDB.Driver/IFindFluent.cs [hunk 1]
```diff
@@ -112,6 +112,9 @@ public interface IFindFluent<TDocument, TProjection> : IAsyncCursorSource<TProje
         /// <summary>
         /// Limits the number of documents.
         /// </summary>
+        /// <remarks>
+        /// Limit equals to zero or null is equivalent to setting no limit.
```
Comment (JamesKovacs) [inline src/MongoDB.Driver/IFindFluent.cs:116]: This is a reasonable request to improve our documentation and is similar to the note in our [shell documentation on limit](https://www.mongodb.com/docs/manual/reference/method/cursor.limit/). The remark would read more naturally as "Limit of zero or null is..."

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Thank you for contributing to the MongoDB .NET/C# Driver by clarifying our documentation. Please implement the minor wording change requested and I will merge your change into our source tree.

File: src/MongoDB.Driver/IFindFluent.cs (hunk 1 above)
Comment (leandromoh) [reply src/MongoDB.Driver/IFindFluent.cs]: adjust done @JamesKovacs ! thanks for review!

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**